Remove unneeded Qt5Widgets_INCLUDE_DIRS reference

This references causes a build path to be saved in a CMake export file:

ERROR: rviz-default-plugins-14.4.1-1-r0 do_package_qa: QA Issue: File /opt/ros/rolling/share/rviz_default_plugins/cmake/rviz_default_pluginsExport.cmake in package rviz-default-plugins-dev contains reference to TMPDIR [buildpaths]

Upstream-Status: Pending

Signed-off-by: Rob Woolley <rob.woolley@windriver.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1bdc96b17..ed8740b95 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -240,7 +240,6 @@ add_library(rviz_default_plugins SHARED
 target_include_directories(rviz_default_plugins PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/${PROJECT_NAME}>
-  ${Qt5Widgets_INCLUDE_DIRS}
 )

 target_link_libraries(rviz_default_plugins PUBLIC
